src/Twig/Component/WishlistDetails/CollectiveActions/ExportActionsComponent.php

```diff
@@ -0,0 +1,188 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sylius\WishlistPlugin\Twig\Component\WishlistDetails\CollectiveActions;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\Common\Collections\Collection;
+use Sylius\Bundle\OrderBundle\Factory\AddToCartCommandFactoryInterface;
+use Sylius\Bundle\UiBundle\Twig\Component\TemplatePropTrait;
+use Sylius\Component\Core\Factory\CartItemFactoryInterface;
+use Sylius\Component\Core\Model\OrderItemInterface;
+use Sylius\Component\Order\Context\CartContextInterface;
+use Sylius\Component\Order\Modifier\OrderItemQuantityModifierInterface;
+use Sylius\TwigHooks\LiveComponent\HookableLiveComponentTrait;
+use Sylius\WishlistPlugin\Command\Wishlist\ExportSelectedProductsFromWishlistToPdf;
+use Sylius\WishlistPlugin\Command\Wishlist\ExportWishlistToCsv;
+use Sylius\WishlistPlugin\Command\Wishlist\WishlistItem;
+use Sylius\WishlistPlugin\Entity\WishlistInterface;
+use Sylius\WishlistPlugin\Repository\WishlistRepositoryInterface;
+use Symfony\Component\HttpFoundation\BinaryFileResponse;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\RequestStack;
+use Symfony\Component\HttpFoundation\ResponseHeaderBag;
+use Symfony\Component\HttpFoundation\Session\Session;
+use Symfony\Component\Messenger\MessageBusInterface;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Contracts\Translation\TranslatorInterface;
+use Symfony\UX\LiveComponent\Attribute\AsLiveComponent;
+use Symfony\UX\LiveComponent\Attribute\LiveAction;
+use Symfony\UX\LiveComponent\Attribute\LiveProp;
+use Symfony\UX\LiveComponent\ComponentToolsTrait;
+use Symfony\UX\LiveComponent\DefaultActionTrait;
+
+#[AsLiveComponent]
+final class ExportActionsComponent
+{
+    use ComponentToolsTrait;
+    use DefaultActionTrait;
+    use TemplatePropTrait;
+    use HookableLiveComponentTrait;
+
+    #[LiveProp]
+    public int $wishlistId;
+
+    #[LiveProp]
+    public string $wishlistName = 'wishlist';
+
+    #[LiveProp(writable: true)]
+    public string $selection = '[]';
```

Comment:
A bit besides the point, but I'm seeing the same decoding logic in multiple places. Is there no way of passing the selected ids as a simple array?